src/alphabets.jl

```diff
@@ -213,3 +213,17 @@ function Base.getindex(A::Alphabet{T}, p::Integer) where T
 
     throw(DomainError("Inversion of $(A.alphabet[-p]) is not defined"))
 end
+
+"""
+    inv(w::Word, A::Alphabet)
+Return the inverse of a word `w` in the context of alphabet `A`.
+"""
+function Base.inv(w::AbstractWord, A::Alphabet)
```

Comment:
Docstring is not consistent with the declaration, I think.

Reply:
thanks, done!